src/adapters/bidderFactory.js

```diff
@@ -176,8 +194,30 @@ export function newBidder(spec) {
       // After all the responses have come back, call done() and
       // register any required usersync pixels.
       const responses = [];
+
       function afterAllResponses(bids) {
         const bidsArray = bids ? (bids[0] ? bids : [bids]) : [];
+        // bidderRequest.end = timestamp();
+        let bidIds = bidsArray.map(bid => {
+          if (bid.requestId) {
+            return bid.requestId.toString();
+          }
+          return ' '
+        });
+
+        let missingBids = bidderRequest.bids.filter(bid => {
+          if (bid.bidId) {
+            return bidIds.indexOf(bid.bidId.toString()) === -1;
```

Comment:
Why are you converting bid ids to strings? Seems inefficient and unnecessary

Reply:
see above